src/yadmark/benchmark/external/apfel_utils.py

```diff
@@ -24,10 +24,18 @@ def load_apfel(theory, observables, pdf="ToyLH", use_external_grid=False):
         theory, observables, pdf, use_external_grid=use_external_grid
     )
 
+    is_polarized = [obs_name.startswith("g") for obs_name in observables["observables"]]
+    is_polarized = np.unique(is_polarized)
+    if is_polarized.size != 1:
+        raise ValueError(
+            "Apfel can't compute polarized and unpolarized observables at the same time."
+        )
+
     # set DIS params
     apfel.SetProcessDIS(observables.get("prDIS", "EM"))
     apfel.SetPropagatorCorrection(observables.get("PropagatorCorrection", 0))
     apfel.SetPolarizationDIS(observables.get("PolarizationDIS", 0))
+    apfel.SetPolarizedEvolution(int(is_polarized))
```

Comment:
```suggestion
    apfel.SetPolarizedEvolution(0 if not is_polarized else 1)
```
I know it's the exact same, but it is not an excellent practice to rely on it. However, this is purely a matter of taste, even more because this behavior [is documented](https://docs.python.org/3/library/stdtypes.html#boolean-values). So, feel free to ignore this comment.

Reply:
And also exaplained [here](https://peps.python.org/pep-0285/). So as long as `bool` inherits from `int` this mapping is perfectly fine, so I'd leave it like this.

Reply:
Okay, but out of curiosity, why do you think this specific casting is not suggested?